ref/src/interpolate.f90

```diff
@@ -38,51 +38,51 @@ subroutine interpolate_allocate(odims, interpFactor, fdims)
     fdims(2) = odims(2) + (odims(2) - odims(1)) * interpFactor
     fdims(4) = odims(4) + (odims(4) - odims(3)) * interpFactor
 
-  end subroutine interpolate_allocate
+  end subroutine interpolate_CalculateSpace
+
+  ! Bilinear interpolation of f with respect to oa.
```

Comment:
@LyndStringer  in the [contributing](https://github.com/NOAA-GSL/SENA-yppm/blob/develop/CONTRIBUTING.md#fortran-style-guide) file, there is an example for using comment header blocks above subroutines.  When the comment blocks are above the subroutines, and consistent, it makes this code significantly easier for me (and others) to follow & read.

Reply:
@robgonzalezpita  - I agree with your request to have this changed.  However, there are a number of style issues in this PR that need fixing.  Given that the style guide was developed while the work for this PR was in-flight, and @LyndStringer has not yet had time to fully digest it, I would like to suggest that we approve/merge this, and then submit another PR with a slew of corrections to bring the code into style compliance. I normally don't like to bypass requirements when evaluating a PR, but I don't want another week to go by with Lynd spending all that time fixing style issues when he is not the expert.

Having said that, I would like to make this a one time thing.  All style requirements will be a prerequisite to approval going forward.

What do you think?

Reply:
@christopherwharrop-noaa I agree. We can address the style issues later. I am a little nit-picky about formatting, but I wanted to bring the styling here to everyone's attention. It also doesn't help that we don't have a linter (or fprettify doesn't adhere to the style guide).

 I don't want to discredit your work @LyndStringer, you've done a great good with this PR!